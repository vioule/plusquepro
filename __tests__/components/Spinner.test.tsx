import { render } from "@testing-library/react";
import Spinner from "@/components/Spinner";

test("render should match snapshot", () => {
  const { container } = render(<Spinner size={4} />);
  expect(container).toMatchSnapshot();
});
