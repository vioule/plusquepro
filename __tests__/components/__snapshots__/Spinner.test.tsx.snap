// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should match snapshot 1`] = `
<div>
  <svg
    height="4em"
    viewBox="0 0 24 24"
    width="4em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      cx="12"
      cy="12"
      fill="currentColor"
      r="0"
    >
      <animate
        attributeName="r"
        begin="0;svgSpinnersPulseMultiple2.end"
        calcMode="spline"
        dur="1.2s"
        fill="freeze"
        id="svgSpinnersPulseMultiple0"
        keySplines=".52,.6,.25,.99"
        values="0;11"
      />
      <animate
        attributeName="opacity"
        begin="0;svgSpinnersPulseMultiple2.end"
        calcMode="spline"
        dur="1.2s"
        fill="freeze"
        keySplines=".52,.6,.25,.99"
        values="1;0"
      />
    </circle>
    <circle
      cx="12"
      cy="12"
      fill="currentColor"
      r="0"
    >
      <animate
        attributeName="r"
        begin="svgSpinnersPulseMultiple0.begin+0.2s"
        calcMode="spline"
        dur="1.2s"
        fill="freeze"
        id="svgSpinnersPulseMultiple1"
        keySplines=".52,.6,.25,.99"
        values="0;11"
      />
      <animate
        attributeName="opacity"
        begin="svgSpinnersPulseMultiple0.begin+0.2s"
        calcMode="spline"
        dur="1.2s"
        fill="freeze"
        keySplines=".52,.6,.25,.99"
        values="1;0"
      />
    </circle>
    <circle
      cx="12"
      cy="12"
      fill="currentColor"
      r="0"
    >
      <animate
        attributeName="r"
        begin="svgSpinnersPulseMultiple0.begin+0.4s"
        calcMode="spline"
        dur="1.2s"
        fill="freeze"
        id="svgSpinnersPulseMultiple2"
        keySplines=".52,.6,.25,.99"
        values="0;11"
      />
      <animate
        attributeName="opacity"
        begin="svgSpinnersPulseMultiple0.begin+0.4s"
        calcMode="spline"
        dur="1.2s"
        fill="freeze"
        keySplines=".52,.6,.25,.99"
        values="1;0"
      />
    </circle>
  </svg>
</div>
`;
